web/src/types/ga-gtag/index.d.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * npm module name:- "ga-gtag"
+ * version:-  "^1.0.1"
+ */
+/**
+ * This can be improved using zulip specific information
+ * on fields that can be passed with this object into the
+ * `install` function.
+ */
+declare type AdditionalConfigInfo = Record<string, unknown>;
+
+// Type reference: https://github.com/idmadj/ga-gtag/blob/6917019745209525c641ac1698d76ff9ebf9a80a/src/index.js
+export declare const install: (
+    trackingId: string,
+    additionalConfigInfo?: AdditionalConfigInfo,
+) => void;
+/**
+ * Improvements can be made using zulip specific information
+ * on values that can be passed as arguments in this function.
+ */
```

Comment:
This comment can be removed now. I think it was referring to the type annotation of `const gtag`, which is now pretty complete as `Gtag.Gtag`.

Reply:
Make sense. Adjusted accordingly. Please check!